### PR TITLE
Get list of product -> Get Global entitlement

### PR DESCRIPTION
**Pull request by Lavishmehta1410**

### CODE REVIEW

File: c1_perf_test_all_scenario.jmx [hunk 1]
```diff
@@ -1,14 +1,14 @@
-<?xml version="1.0"?>
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (piyush-mehta) [inline c1_perf_test_all_scenario.jmx:1]: @Lavishmehta1410  - please remove this file. it is not part of this PR.

**Comment by Lavishmehta1410** [comment]
This is of no significance anymore as we have picked the individual apis for further updates. So this will be covered as a part of that activity
